js/src/classic-editor.js

```diff
@@ -216,6 +207,31 @@ jQuery(
 			}
 		);
 
+		// Listen to `onPostLangChoice` to perform actions after the language has been changed.
+		document.addEventListener(
+			'onPostLangChoice',
+			( e ) => {
+				// Update the old language with the new one to be able to compare it in the next changing.
+				initializeLanguageOldValue();
+				// modifies the language in the tag cloud
+				$( '.tagcloud-link' ).each(
+					function () {
+						var id = $( this ).attr( 'id' );
+						tagBox.get( id );
+					}
+				);
+
+				let dir = e.detail.lang.is_rtl ? 'rtl' : 'ltr'
+
+				// Modifies the text direction
+				$( 'body' ).removeClass( 'pll-dir-rtl' ).removeClass( 'pll-dir-ltr' ).addClass( 'pll-dir-' + dir );
+				$( '#content_ifr' ).contents().find( 'html' ).attr( 'lang', e.detail.lang.locale ).attr( 'dir', dir );
+				$( '#content_ifr' ).contents().find( 'body' ).attr( 'dir', dir );
+
+				pll.media.resetAllAttachmentsCollections();
```

Comment:
This is another action which would benefit a new comment.

Reply:
I'm not sure to understand, do you want me to add a comment to say what this line does for the media?